src/sentry/snuba/metrics/query_builder.py

```diff
@@ -367,6 +368,13 @@ def translate_meta_results(
     for record in meta:
         operation, column_name = parse_expression(record["name"])
 
+        metric_field = alias_to_metric_field.get(record["name"], None)
+        # If we have a percentile operation then we want to convert its type from Array(float64) to Float64
+        # because once we receive a percentile result from ClickHouse we automatically pop from the array the
+        # first element thus the datatype itself must also be changed in the metadata.
+        if metric_field and metric_field.op and metric_field.op in OPERATIONS_PERCENTILES:
+            record["type"] = "Float64"
```

Comment:
Lets handle this not silence it
```suggestion
        try:
             metric_field = alias_to_metric_field[record["name"]]
         except KeyError:
             raise InvalidParams(f"Field {record['name']} was not in the select clause")
        # If we have a percentile operation then we want to convert its type from Array(float64) to Float64
        # because once we receive a percentile result from ClickHouse we automatically pop from the array the
        # first element thus the datatype itself must also be changed in the metadata.
        if metric_field and metric_field.op and metric_field.op in OPERATIONS_PERCENTILES:
            record["type"] = "Float64"
```